website/content/docs/commands/job/start.mdx

```diff
@@ -0,0 +1,85 @@
+Use the `nomad job start` command to start the latest version of an existing job
+with a status of Stopped. Refer to [Job statuses] for status explanations.
+
+If your Nomad servers use [consul service identity] authentication and your job
+uses Consul Connect services, you must supply a Consul token. Nomad does not
+persist the Consul token when you [run] a job. Use the `-consul-token` flag to
+pass the token to the `nomad job start` command. Alternately, you may use the
+`$CONSUL_HTTP_TOKEN` environment variable.
+
+If your Nomad servers require [Vault policy] authentication, you must supply a
+Vault token. Nomad does not persist the Vault token when you [run] a job.
+Use the `-vault-token` flag to pass the token to the `nomad job start` command.
+Alternately, you may use the `$VAULT_TOKEN` environment variable.
+
+## Usage
+
+```shell-session
+nomad job start [options] <job_id>
+```
+
+The `nomad job start` command requires the job ID. Nomad starts the latest
+version of the job. This command fails if the job does is not currently stopped.
+
+When Nomad uses ACLs, the `nomad job start` command requires a token with one of
+the following capabilities based on the specific scenario:
+
+- The `submit-job`capability for the job's namespace
+- The `list-jobs` capability to run the command with a job prefix instead of the
+  exact job ID
+- The `read-job` capability is required to monitor the resulting evaluation when
+  you do not use the `-detach`option.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Start Options
+
+- `-detach`: Return immediately instead of monitoring. Nomad outputs a new
+  evaluation ID, which you can use to examine the evaluation using the
+  [eval status] command.
+
+- `-consul-token`: If set, Nomad sends the Consul token with the
+  request. This overrides the token found in the `$CONSUL_HTTP_TOKEN`
+  environment variable.
+
+- `-vault-token`: If set, Nomad includes the Vault token with the request. This
+  overrides the token found in the `$VAULT_TOKEN` environment variable.
+
+- `-verbose`: Show full information.
+
+## Examples
+
+Start a previously stopped job.
+
+```shell-session
+$ nomad job status
+ID       Type     Priority  Status          Submit Date
+example  service  50        dead (stopped)  2025-02-11T15:33:27-05:00
+
+$ nomad job start example
+==> 2025-02-11T15:34:48-05:00: Monitoring evaluation "8b715538"
+    2025-02-11T15:34:48-05:00: Evaluation triggered by job "example"
+    2025-02-11T15:34:49-05:00: Evaluation within deployment: "866ca498"
+    2025-02-11T15:34:49-05:00: Allocation "4d050576" created: node "8e17fa4d", group "echo"
+    2025-02-11T15:34:49-05:00: Evaluation status changed: "pending" -> "complete"
+==> 2025-02-11T15:34:49-05:00: Evaluation "8b715538" finished with status "complete"
+```
+
+[eval status]: /nomad/docs/commands/eval/status
+[consul service identity]: /nomad/docs/configuration/consul#allow_unauthenticated
```

Comment:
`/nomad/docs/configuration/consul#allow_unauthenticated` I don't see the `allow_unauthenticated` parameter on this page.